src/cf-core/usage.md

```diff
@@ -535,65 +561,84 @@ Modify link properties using the following mixins.
 
 ##### Link colors
 
-Calling the mixin without arguments will set the following states - default(pacific), :hover(pacific-50), :focus:(pacific), :visited teal, :active navy.
+Calling the mixin without arguments will set the following states:
+`default` - `@pacific`, `:hover` - `@pacific-50`, `focus:` - `@pacific`,
+`:visited` - `@teal`, `:active` - `@navy`.
+
+[//]: # (NOTE: These aren't the default colors within this project, only once the brand theme has been applied.)
```

Comment:
Yeah, I wasn't sure if that'd be more or less helpful since we don't document the variables names anywhere else, just the [full names here](https://cfpb.github.io/design-manual/page-components/links.html). Another reason to theme the project on-brand.